src/main/java/com/moneytransfer/service/CurrencyExchangeServiceImpl.java

```diff
@@ -23,14 +23,23 @@ public class CurrencyExchangeServiceImpl implements CurrencyExchangeService {
     @Value("${freecurrencyapi.apiUrl}")
```

Comment:
For custom application properties you can create your application namespace ie:
application.freecurrencyapi.host